docs/cli-reference/ark_schedule_describe.md

```diff
@@ -0,0 +1,39 @@
+## ark schedule describe
+
+Describe schedules
+
+### Synopsis
+
+
+Describe schedules
```

Comment:
Should this be singular? Looks like the rest of the file is.

Reply:
Which?

Reply:
L1, L11, and L38 both use the singular.

I think this is a result of me reading too fast on first pass; `ark schedule describe -l <selector` will fetch the matched schedules and describe them. I was thinking it would describe a single schedule instead of a list.

Reply:
These docs are autogenerated. We can control certain aspects, such as `Describe schedules` and `Work with schedules`. Do you think something needs to change?

Reply:
I see now; I don't think I have a suggestion for changing this. Didn't realize that these were autogenerated.